### PR TITLE
Buzzer

### PR DESCRIPTION
**Pull request by Aksel147**
Buzzer implementation with unit tests + updated unit and integration tests to succeed with buzzer

### CODE REVIEW

File: Microwave.App/Program.cs [hunk 1]
```diff
@@ -38,6 +40,8 @@ static void Main(string[] args)
             timeButton.Press();
 
             startCancelButton.Press();
+            
+            buzzer.BuzzThreeTimes();
```
Comment (Nicolai-Bak) [inline Microwave.App/Program.cs:44]: Skal fjernes igen

**Review by Nicolai-Bak — CHANGES_REQUESTED** [review]
Se kommentar i program.cs

File: Microwave.App/Program.cs (hunk 1 above)
Comment (Aksel147) [reply Microwave.App/Program.cs]: Sådan

**Review by Nicolai-Bak — APPROVED** [review]
Nu ser det godt ud

**Pull request merged**